content/kubermatic/main/installation/upgrading/upgrade-from-2.27-to-2.28/_index.en.md

```diff
@@ -150,7 +150,7 @@ Administrators are advised to begin migrating to the new chart as soon as possib
 #### Migration Procedure
 
 {{% notice warning %}}
-After migrating to Dex, users may encounter login issues due to invalid tokens. To resolve, clear browser cookies for the application domain and log in again.
+After migrating to Dex, users may encounter login issues due to invalid tokens. To resolve, clear browser cookies for the application domain and log in again. Additionally, restart the `kubermatic-api` to ensure proper initialization and synchronization with the new Dex configuration.
 {{% /notice %}}
```

Comment:
A warning like this doesn't suffice since if this is really required then it should be an explicit step in the migration process.